apps/eew/sceewlog/descriptions/sceewlog.xml

```diff
@@ -232,6 +232,11 @@
 					Set the magnitude authors in a list comma separated. This should be generally the app name followed by the hostname.
 					</description>
 				</parameter>
+				<parameter name="stationMagNumber" type="string" default="Mfd:5,MVS:4,Mlv:5">
```

Comment:
Okay, I am removing the default values for Mlv since this is not used. Just one comment for the default value of MVS. I saw some fake events at INETER with only one MVS quantified in a station. I would say at least 2 stations(?)

Reply:
I set your suggestions for the default values. At the end this is on user's decision or preference. I just thought in a particular case but this does not apply for all. Can you please merge it if this is fine?